Readme.md

```diff
@@ -18,171 +18,444 @@ This is an ipm package, which contains one or more reusable assets within the ip
+<a name="BigQuery..Project..Dataset"></a>
+
+#### Project~Dataset(datasetID) ⇒ <code>Object</code>
+**Kind**: inner method of [<code>Project</code>](#BigQuery..Project)  
+**Returns**: <code>Object</code> - - consists of Table Object and other methods for Dataset Object  
+
+| Param | Type |
+| --- | --- |
+| datasetID | <code>string</code> | 
+
+
+* [~Dataset(datasetID)](#BigQuery..Project..Dataset) ⇒ <code>Object</code>
+    * [~Table(tableID)](#BigQuery..Project..Dataset..Table) ⇒ <code>Object</code>
+        * [~InsertAll(requestBody, callback)](#BigQuery..Project..Dataset..Table..InsertAll)
+    * [~Get(callback)](#BigQuery..Project..Dataset..Get) ⇒
+    * [~Delete(callback)](#BigQuery..Project..Dataset..Delete)
+    * [~Insert(requestBody, callback)](#BigQuery..Project..Dataset..Insert) ⇒
+    * [~Update(requestBody, callback)](#BigQuery..Project..Dataset..Update)
+    * [~List(callback)](#BigQuery..Project..Dataset..List)
+
+<a name="BigQuery..Project..Dataset..Table"></a>
+
+##### Dataset~Table(tableID) ⇒ <code>Object</code>
+**Kind**: inner method of [<code>Dataset</code>](#BigQuery..Project..Dataset)  
+**Returns**: <code>Object</code> - - consists of methods for the Table Object  
+
+| Param | Type |
+| --- | --- |
+| tableID | <code>string</code> | 
+
+<a name="BigQuery..Project..Dataset..Table..InsertAll"></a>
+
+###### Table~InsertAll(requestBody, callback)
+Tabledata: insertAll
+insertAll - Streams data into BigQuery one record at a time without needing to run a load job.
 
-For more information https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/delete
+For more Information regarding optional parameters and response structure: 
+https://cloud.google.com/bigquery/docs/reference/rest/v2/tabledata/insertAll
 
-**Kind**: instance method of [<code>BigQuery</code>](#BigQuery)  
+**Kind**: inner method of [<code>Table</code>](#BigQuery..Project..Dataset..Table)  
 
 | Param | Type | Description |
 | --- | --- | --- |
-| projectID | <code>string</code> | name of the project |
-| datasetID | <code>string</code> | name of the dataset you want to delete in the project |
-| callback | <code>callback</code> | callback with function signature: (err, data) |
+| requestBody | <code>Object</code> | {                         "kind": "bigquery#tableDataInsertAllRequest",                         "skipInvalidRows": boolean,                         "ignoreUnknownValues": boolean,                         "templateSuffix": string,                         "rows": [                             {                                 "insertId": string,                                 "json": {                                     (key): (value)                                 }                             }                         ]                     } |
+| callback | <code>callback</code> |  |
 
 **Example**  
 ```js
-response
-If successful, this method returns an empty response body.
+:- Inside of a ClearBlade service, after the initialization
```

Comment:
This is not js syntax

Reply:
removed, assuming the user is going to write code inside a ClearBlade service